src/Tribe/Tickets_Handler.php

```diff
@@ -880,9 +880,13 @@ public function get_ticket_totals( $ticket ) {
 
 		$totals = array_map( 'intval', $totals );
 
-		// Remove Pending from total
+		// Remove Pending from total.
 		$totals['sold'] -= $totals['pending'];
 
+		// Update Stock.
+		$totals['stock'] = tribe_tickets_get_capacity( $ticket->ID ) - $totals['sold'];
```

Comment:
`tribe_tickets_get_capacity` should calculate RSVPs as well. So i believe the logic is not right.

Additionally shouldn't we subtract `$totals['pending']` as well since they are no longer in the stock ?

Reply:
This is the `get_ticket_totals()` functions, it works for RSVPs as well.

The 'pending' have been subtracted in the line above, or am I missing something?

Reply:
Yes because they have been substracted from the `$totals['sold']` they need to be substracted from the `$totals['stock']` again or move line 884 after 887